cobbler/modules/sync_post_wingen.py

```diff
@@ -446,15 +446,19 @@ def gen_win_files(distro, meta):
                 pi_file = tempfile.NamedTemporaryFile()
                 pi_file.write(bytes(data, "utf-8"))
                 pi_file.flush()
-                cmd = [
-                    "/usr/bin/wimdir %s 1 | /usr/bin/grep -i '^/Windows/System32/startnet.cmd$'"
-                    % ps_file_name
-                ]
-                startnet_path = utils.subprocess_get(cmd, shell=True)[0:-1]
+                cmd = ["/usr/bin/wimdir", ps_file_name, "1"]
+                wimdir_result = utils.subprocess_get(cmd, shell=False)
+                wimdir_file_list = wimdir_result.split("\n")
+                # grep for /Windows/System32/startnet.cmd
+                startnet_path = "/Windows/System32/startnet.cmd"
+
+                if startnet_path not in wimdir_file_list:
+                    raise FileNotFoundError("Windows startnet executable missing!")
```

Comment:
`grep` must be with the `-i` option. We need to know the real name of the `startnet_path` file, case-insensitive. It can be /**W**indows/**S**ystem32/startnet.cmd or /**w**indows/**S**ystem32/startnet.cmd or /**W**indows/**s**ystem32/startnet.cmd etc.
```suggestion
                # grep -i for /Windows/System32/startnet.cmd
                startnet_path = "/Windows/System32/startnet.cmd"

                for x in wimdir_file_list:
                    if x.lower() == startnet_path.lower():
                        startnet_path = x
```